src/wp-admin/includes/template.php

```diff
@@ -609,9 +609,9 @@ function list_meta( $meta ) {
  *
  * @since 2.5.0
  *
- * @param array $entry
- * @param int   $count
- * @return string
+ * @param array $entry Array of meta data.
+ * @param int   $count Counter variable passed by reference.
+ * @return string      Single row of public meta data.
```

Comment:
```suggestion
 * @param array $entry An array of meta data keyed on 'meta_key' and 'meta_value'.
 * @param int   $count Reference to the row number.
 * @return string A single row of public meta data.
```
Only `@param` annotations should be aligned with each other. `@return` should have a single space.